### PR TITLE
Web monitoring service

### PR DESCRIPTION
**Pull request by shubhamgavali1998**
## What does this PR do?

# Monitoring package

- [ ] Bugfix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist before merging

_Put an `x` in the boxes that apply_

- [ ] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

File: packages/monitoring/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@groww-tech/monitoring",
+  "version": "0.0.1",
+  "description": "Web Monitoring is a service used to inject prometheus monitoring configuration",
+  "main": "src/index.ts",
```
Comment (tusharsehwagg) [inline packages/monitoring/package.json:5]: serve it from dist folder and add module and types too. Take reference from ella.

File: packages/monitoring/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/Groww/webster/issues"
+  },
+  "homepage": "https://github.com/Groww/webster/tree/develop/packages/monitoring#readme",
+  "devDependencies": {
+    "@groww-tech/ella": "0.2.4",
```
Comment (tusharsehwagg) [inline packages/monitoring/package.json:43]: ella is a dev dependency?

File: packages/monitoring/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+import express, { Request, Response } from 'express';
+
+import client from 'prom-client';
```
Comment (tusharsehwagg) [inline packages/monitoring/src/index.ts:4]: can name it promClient or more better prometheusClient

File: packages/monitoring/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+register.registerMetric(httpRequestTimer);
+
+
+/**
+ * A function to initialize the prometheus service. It will monitor the request duration by storing route, status code and request method of each request.
+ * @param PROMETHEUS_PORT Port which prometheus will listen
+ * @param routeToExpose Route on which server will expose the metrics
+ * @returns void
+ */
+export function initPrometheus(PROMETHEUS_PORT: number, routeToExpose: string) {
```
Comment (tusharsehwagg) [inline packages/monitoring/src/index.ts:45]: as per the example provided above in readme, second param should be optional either or should update in readme too.

**Review by tusharsehwagg — CHANGES_REQUESTED** [review]

File: packages/monitoring/package.json (hunk 2 above)
Comment (shubhamgavali1998) [reply packages/monitoring/package.json]: removed ella, it was not getting used

**Review by abhishekjhagroww — APPROVED** [review]

**Review by tusharsehwagg — APPROVED** [review]

**Pull request merged**